src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaInspector.tsx

```diff
@@ -162,7 +164,7 @@ const SchemaInspector = ((props: ISchemaInspectorProps) => {
             className={classes.navButton}
             onClick={onGoToDefButtonClick}
           >
-            Gå til hovedkomponent
+            {getTranslation('schema_editor.go_to_main_component', props.language)}
```

Comment:
Why have the use of both  `getTranslation` and the shared util method `getLanguageFromKey`? 

Does the schema editor not have any bindings to these shared utils? If so I guess it makes sense.

Reply:
Yes, schema-editor does not have access to the shared utils. And I didn't want to name the method the same to create confusion when doing imports.